recipe/meta.yaml

```
@@ -1,19 +1,19 @@
 {% set name = "arcticdb" %}
-{% set version = "4.0.0rc1" %}
+{% set version = "4.0.0rc2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/man-group/ArcticDB/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 3e2bbd667f1cf198956219590752476bb59aecfd11e580a34f19340debe33401
+  sha256: 93ad9ee68b52e2b7a373ffbb5903d6cc833afc854811f8f0c24777e2fe0b60fc
```

As reported on the CI, I also have checked locally.
```suggestion
  sha256: ed2b8c291e740a9f80372349e16a6c3e584433130513529e6d878b66e28317c6
```